tools-local/unix/invoke-extension.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+
+# Locates and invokes the given extension script, which does a search or acquire, with the specified arguments. 
+
+usage()
+{
+    echo "Usage: $0 ScriptName ToolName -StrictToolVersionMatch --OverrideScriptsFolderPath <path>"
```

Comment:
Is there a reference that we should follow. For example, http://courses.cms.caltech.edu/cs11/material/general/usage.html